ui/raidboss/data/06-ew/dungeon/aloalo_island.ts

```diff
@@ -187,71 +155,70 @@ const triggerSet: TriggerSet<Data> = {
       response: Responses.getOutThenIn(),
     },
     {
-      id: 'Aloalo Ketuduke Roar',
+      id: 'Aloalo Ketuduke Fluke Typhoon',
       type: 'StartsUsing',
-      netRegex: { id: '8A92', source: 'Ketuduke', capture: false },
+      netRegex: { id: '8A84', source: 'Ketuduke', capture: false },
       infoText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Adds',
-          de: 'Adds',
-          ja: 'ざこギミック',
+          en: 'Bubbles Move 2 Tiles',
+          de: 'Blasen 2 Flächen Rückstoß',
+          ja: '泡のみ2マスのノックバック',
         },
       },
     },
     {
-      id: 'Aloalo Ketuduke Fluke Typhoon',
-      type: 'StartsUsing',
-      netRegex: { id: '8A84', source: 'Ketuduke', capture: false },
+      // Path 01
+      id: 'Aloalo Ketuduke Water III',
+      type: 'Tether',
+      netRegex: { id: '0001', source: 'Summoned Apa' },
+      condition: Conditions.targetIsYou(),
       infoText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Bubbles 2 tiles knockback',
-          de: 'Blasen 2 Flächen Rückstoß',
-          ja: '泡のみ2マスのノックバック',
+          en: 'Stretch Tether',
         },
       },
     },
     {
-      id: 'Aloalo Ketuduke Updraft',
+      // Path 02
+      id: 'Aloalo Ketuduke Tidal Wave',
       type: 'StartsUsing',
-      netRegex: { id: '8D0F', source: 'Ketuduke', capture: false },
-      infoText: (_data, _matches, output) => output.text!(),
+      netRegex: { id: '8D12', source: 'Ketuduke', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Bubbles float',
```

Comment:
I changed this trigger into separate triggers with more specific depending on the path you're on.